packages/compass-e2e-tests/helpers/commands/wait-for-connection-result.ts

```diff
@@ -37,9 +40,11 @@ export async function waitForConnectionResult(
   }
 
   if (TEST_MULTIPLE_CONNECTIONS) {
-    await browser
-      .$(Selectors.ConnectionModal)
-      .waitForDisplayed({ reverse: true });
+    if (fromForm) {
```

Comment:
do we need to skip this? does it error without the form, or do we have cases when we want to keep the form open while connecting from the sidebar? asking because I find the `fromForm` parameter a bit confusing, so I'd rather avoid it if possible